apps/pollbook/frontend/src/app_election_manager_screen.test.tsx

```diff
@@ -0,0 +1,111 @@
+import { expect, test, beforeEach, afterEach, vi } from 'vitest';
```

Comment:
should this be in election_manager_screen.test or should that be renamed to app_election_manager_screen.test? I guess you have different beforeEach steps here but I'm not really following the distinction between the test files.

Reply:
The `app_<role>_screen` tests predate the individual component tests. I ported this test with as little diff as possible. The main difference is in this test we render the `<App>` component and in `election_manager_screen.test.tsx` we render the `ElectionManagerScreen` component. `app.tsx` is a pretty thin wrapper on these screens, basically just determining auth and rendering appropriately, so there's not much value added for the "e2e" tests. It's a good idea to just move this test into `election_manager_screen.test.tsx`.

Reply:
Something about the component test doesn't work well with fake timers, so going to leave this in its own file unless we have a compelling reason to combine that outweighs the cost of investigating the fake timer issue.